File: sample-videochat-java/app/src/main/java/com/quickblox/sample/videochat/java/adapters/ReconnectingUserModel.java
package com.quickblox.sample.videochat.java.adapters;

import com.quickblox.users.model.QBUser;

public class ReconnectingUserModel {
    private QBUser user;
    private String reconnectingState;

    public ReconnectingUserModel(QBUser user, String reconnectingState) {
        this.user = user;
        this.reconnectingState = reconnectingState;
    }

    public QBUser getUser() {
        return user;
    }

    public String getReconnectingState() {
        return reconnectingState;
    }

    public void setUser(QBUser user) {
        this.user = user;
    }

    public void setReconnectingState(String reconnectingState) {
        this.reconnectingState = reconnectingState;
    }
}

File: sample-videochat-kotlin/app/src/main/java/com/quickblox/sample/videochat/kotlin/adapters/ReconnectingUserModel.kt
package com.quickblox.sample.videochat.kotlin.adapters

import com.quickblox.users.model.QBUser


class ReconnectingUserModel(private var user: QBUser, private var reconnectingState: String) {
    fun getUser(): QBUser {
        return user
    }

    fun getReconnectingState(): String {
        return reconnectingState
    }

    fun setUser(user: QBUser) {
        this.user = user
    }

    fun setReconnectingState(reconnectingState: String) {
        this.reconnectingState = reconnectingState
    }
}